### PR TITLE
(sysinternals) `ShellRunAs.exe` Update NonConforming Reg Key In helpers

### LINKED ISSUE #1777: (sysinternals) `ShellRunAs.exe` Prompted For Eula
## Expected Behavior

I'd prefer to accept all Eula's during install.
## Current Behavior

I was asked to accept the Eula when running `ShellRunAs.exe`
## Possible Solution

The registry key added was `ShellRunas - Sysinternals: www.sysinternals.com` at path of `HKCU:\SOFTWARE\SysInternals\` with `EulaAccepted` value at `1`. [Here](https://www.hexacorn.com/blog/2014/11/30/sysinternals-eulagoogoolizer/) is an article discussing in more detail.
## Steps to Reproduce (for bugs)

1. Install sysinternals fresh
2. Run `ShellRunAs.exe` without any flags.

## Context

It was a minor inconvenience.
## Your Environment

* Package Version used: Unknown. I have the latest sysinternals version and used `choco upgrade`.
* Operating System and version: `Windows 10 1909 ( OS Build 18363.1377 )`
* Chocolatey version:`0.11.3`
* Install/uninstall gist:

### PR DESCRIPTION
**Pull request by mattcargile**
## Description

Add variable with long key name.
Include variable within the `$tools` array.
## Motivation and Context

Fixes #1777 
## How Has this Been Tested?

Have not tested the changes as far as chocolatey packaging goes. I tested the registry key Eula Change.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [ ] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/sysinternals/tools/helpers.ps1
Comment (TheCakeIsNaOH) [inline automatic/sysinternals/tools/helpers.ps1:3]: What is the reason you have this as a separate variable instead of inlining it like the other ones are?

Comment (mattcargile) [reply automatic/sysinternals/tools/helpers.ps1:3]: It was to ease the formatting of the array. I couldn't quite get satisfied trying to keep the array columns in sync and I wanted to try to make the diff easier to read.

Comment (TheCakeIsNaOH) [reply automatic/sysinternals/tools/helpers.ps1:3]: Fair enough, maybe someone else should take a look at the formatting.

Comment (mattcargile) [reply automatic/sysinternals/tools/helpers.ps1:3]: Gotcha. I don't mind in-lining it in the array and doing the spacing work if that would be more ideal. We could also add an `if` block within the `foreach` but I thought that would be too much extra code.

File: automatic/sysinternals/tools/helpers.ps1
Comment (AdmiringWorm) [inline automatic/sysinternals/tools/helpers.ps1:15]: Although not really necessary, it would still be better to quote the variable just to be sure.

```suggestion
"$ShellRunas",        "SigCheck",                  "Streams",    "Strings",         "Sync",
```

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]
Just a single thing to update, otherwise this looks fine to me.

It still however needs to be tested in the [chocolatey-test-environment](https://github.com/chocolatey-community/chocolatey-test-environment) before we can merge this in.

**Comment by mattcargile** [comment]
NOTE: LogonSessions needs to be added. It is one of the tools `LogonSessions.exe`.

**Pull request closed without merging**
